cfme/automate/explorer/method.py

```diff
@@ -484,6 +507,23 @@ def create(
                 'playbook_input_parameters': playbook_input_parameters
             })
             validate = False
+
+        if location == 'Ansible Tower Workflow Template':
+            add_page.fill({
+                'method_name': name,
+                'method_display_name': display_name,
+                'ansible_max_ttl': max_ttl,
+            })
+            validate = False
+
+        if location == 'Ansible Tower Job Template':
```

Comment:
I don't see any difference in add_page.fill for both the locations , why do we need `if` checks ?